persistent/Database/Persist/Sql/Types.hs

```diff
@@ -56,6 +59,16 @@ type Migration = WriterT [Text] (WriterT CautiousMigration (ReaderT SqlBackend I
 
 type ConnectionPool = Pool SqlBackend
 
+data ConnectionPoolConfig = ConnectionPoolConfig
+    { connectionPoolConfigStripes :: Int
+    , connectionPoolConfigIdleTimeout :: NominalDiffTime
+    , connectionPoolConfigSize :: Int
```

Comment:
I've tried to keep the order of stripes + idle timeout + pool size consistent. I think it's somewhat unfortunate that it's easy to mix up stripes and pool size though.